gaztec/README.md

```diff
@@ -0,0 +1,50 @@
+# React + TypeScript + Vite
+
+This template provides a minimal setup to get React working in Vite with HMR and some ESLint rules.
+
```

Comment:
This is autogenerated no? It'd be good to replace it with 1) what this is, and 2) how to run it

Reply:
Done!